docs/using-external-configuration-secrets.md

```diff
@@ -85,9 +85,11 @@ metadata:
   name: <secret-name>-chatbot-configuration
   namespace: <target-namespace>
 stringData:
+  chatbot_default_provider: <LLM provider type (rhoai, openai, or azure-openai)>
   chatbot_url: <Chatbot LLM URL>
-  chatbot_model: <Chatbot model name>
-  chatbot_token: <Chatbot LLM access token>
+  chatbot_model: <Chatbot model name or deployment name>
+  chatbot_token: <Chatbot LLM access token or API key>
+  chatbot_model_config_extras: <JSON dict with provider-specific config (optional)>
```

Comment:
`ansible-ai-connect-operator` already supports two "extra" settings: `postgres_extra_settings` and `extra_settings`. They are defined as an array of objects, whicha are dicts that have  `setting` and `value` as keys.  I think it is more consistent to define this as `chatbot_model_extra_settings` with the same type of dicts as elements.

Reply:
@TamiTakamiya  I did try to use the `chatbot_model_config_extras` as `Dict`, but it seems the secrets do not support it, so that's why I'm using json
check the error:
```
➜  crc oc apply -f chatbot-config-secret.yml
Error from server (BadRequest): error when creating "chatbot-config-secret.yml": Secret in version "v1" cannot be handled as a Secret: json: cannot unmarshal array into Go struct field Secret.stringData of type string
```

how I tried
```yaml
...
  chatbot_model_config_extras:
    - api_version: 2025-01-01-preview
```

Reply:
I see. I am closing this conversation as resolved.